.github/workflows/linux_meson.yml

```diff
@@ -102,3 +102,111 @@ jobs:
       run: |
         export OMP_NUM_THREADS=2
         python dev.py -n -j 2
+
+  prerelease_deps_coverage_64bit_blas:
+    name: Prerelease deps and Blas 64bit
+    # If using act to run CI locally the github object does not exist and the usual skipping should not be enforced
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
I did some minimal changes in. The main goal being to move the job and make it not crash all the time.

There is a lot to do there, but we need to have a look at the CI matrix in general before changing things like that.

Reply:
> There is a lot to do there, but we need to have a look at the CI matrix in general before changing things like that

I agree. We can address these in a follow-up. I thought it would be nice to start a discussion.

Reply:
Ah sure 😃